### PR TITLE
a debian package for bundle service

### PR DESCRIPTION
**Pull request by jrwren**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,44 @@
+export GOPATH = ${CURDIR}
+PROJECT := github.com/juju/bundleservice
+BRANCH ?= master
+
+SOURCE_CHANGES := ../bundleservice_$(shell dpkg-parsechangelog -SVersion)_source.changes
+
+$(GOPATH)/bin/godeps:
+	go get -v launchpad.net/godeps
```
Comment (bac) [inline Makefile]: No -- use the one from github.com/rogpeppe/godeps please.

**Comment by bac** [comment]
:+1: with one rant.  :)

**Comment by jrwren** [comment]
:shipit:

**Comment by jujugui** [comment]
Merged build finished. Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice-deb/1/
Test FAILed.

**Comment by mhilton** [comment]
:+1:

**Comment by jujugui** [comment]
Merged build finished. Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice-deb/2/
Test FAILed.

**Comment by jujugui** [comment]
Merged build finished. Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice-deb/3/
Test FAILed.

**Comment by jujugui** [comment]
Merged build finished. Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice-deb/7/
Test PASSed.

**Comment by jrwren** [comment]
:shipit:

**Comment by jrwren** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/bundleservice-deb-merge

**Pull request merged**